apps/backoffice-v2/src/common/components/templates/Providers/Providers.tsx

```diff
@@ -3,23 +3,21 @@ import { FunctionComponent, PropsWithChildren } from 'react';
 import { queryClient } from '../../../../lib/react-query/query-client';
 import { AuthProvider } from '../../../../domains/auth/context/AuthProvider/AuthProvider';
 import { env } from '../../../env/env';
-// import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
+import { useLocation } from 'react-router-dom';
 
 export const Providers: FunctionComponent<PropsWithChildren> = ({ children }) => {
+  const { state } = useLocation();
+
   return (
     <QueryClientProvider client={queryClient}>
       <AuthProvider
-        protectedRoutes={
-          [
-            // Uses String.prototype.startsWith until authenticated layout is implemented.
-            '/en/case-management/',
-          ] as const
-        }
         redirectAuthenticatedTo={'/en/case-management/entities'}
         redirectUnauthenticatedTo={'/en/auth/sign-in'}
         signInOptions={{
           redirect: env.VITE_AUTH_ENABLED,
-          callbackUrl: '/en/case-management/entities',
+          callbackUrl: state?.from
```

Comment:
Redirects users to the last page they were on when signed in. In future iterations we can recover state like filters, sort, and pagination.